pandas/_libs/window/aggregations.pyx

```diff
@@ -56,7 +56,7 @@ cdef:
 cdef inline int int_max(int a, int b): return a if a >= b else b
 cdef inline int int_min(int a, int b): return a if a <= b else b
 
-cdef inline bint is_monotonic_start_end_bounds(
+cdef bint is_monotonic_start_end_bounds(
```

Comment:
hmm, why can't this *also* be in-line?

Reply:
This was the cause of the `warning: pandas/_libs/window/aggregations.pyx:60:4: Buffer unpacking not optimized away.` warning. @mroeschke suggested a potential inline approach that got rid of the warning and used memory views but caused test failures. Could still adjust a few things to make that work but I think without a larger effort this is the best option for now

Reply:
this is a highly optimized path

if u can show this doesn’t have any perf regressions then ok